js/table/tableview.ts

```diff
@@ -584,6 +594,15 @@ export class TableView {
             return (currentPlayer.Status() & 1) !== 0;
         }, this);
 
+        this.isSitOut.subscribe((value) => {
```

Comment:
Should this be `this.actionBlock.isSitOut` ?

Reply:
Looks like whole moving of `suppressSetSitoutStatus` is unnescessary, since it is sould belongs to ActionBlock and it should be private, since this is implementation details.

Reply:
No

Reply:
I see that this variable related somehow to the fact that we have panel work in sync with `skipDeals` checkbox.